doc/examples/data/plot_3d.py

```diff
@@ -0,0 +1,25 @@
+"""
+==========================================
+datasets with 3 or more spatial dimensions
+==========================================
+
+Most scikit-image functions are compatible with 3-D datasets, ie images with
+3 spatial dimensions (as opposed to 2-D multichannel arrays, which also have
```

Comment:
```suggestion
3 spatial dimensions (to be distinguished from 2D multichannel images, which are also arrays with
```
Sorry for breaking the line length...